substrate/frame/referenda/src/lib.rs

```diff
@@ -901,8 +946,8 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		let alarm_interval = T::AlarmInterval::get().max(One::one());
 		// Alarm must go off no earlier than `when`.
 		// This rounds `when` upwards to the next multiple of `alarm_interval`.
-		let when = (when.saturating_add(alarm_interval.saturating_sub(One::one())) /
-			alarm_interval)
+		let when = (when.saturating_add(alarm_interval.saturating_sub(One::one()))
```

Comment:
My auto-formatter does the same but I've received comments before about not changing this.